packages/tools/pie-icons-configs/configs-react.js

```diff
@@ -0,0 +1,39 @@
+import {
+    regularIconSizes, largeIconSizeModule, largeIconSizeDefault, validateGetLargeIconSize, validateGetRegularIconClass,
+} from './configs.js'; // eslint-disable-line import/extensions
+
+/**
+ * Returns props for the svg element based on the provided parameters
+ * @param {string} svgClasses - String of classes assigned to the icon
+ * @param {string} staticClass - String of classes assigned to the component
```

Comment:
Assuming `staticClass` is just singular here?

```suggestion
 * @param {string} staticClass - Classname String assigned to the component
```